sklearn/ensemble/_weight_boosting.py

```diff
@@ -371,7 +372,7 @@ class AdaBoostClassifier(ClassifierMixin, BaseWeightBoosting):
         deviance loss function. Binary classification is a special case where
         only a single regression tree is induced.
 
-    sklearn.tree.DecisionTreeClassifier
+    DecisionTreeClassifier
```

Comment:
This still needs to be reverted because `DecisionTreeClassifier` is not a part of the `sklearn.ensemble` module.